build_system/src/main.rs

```diff
@@ -22,6 +23,7 @@ fn usage() {
         "\
 Available commands for build_system:
 
+    clean    : Run clean command
```

Comment:
I think I'll keep the name `clean-all` as I could easily see new commands added like `clean-projects`.

Reply:
Wouldn't you prefer instead having `clean all` and `clean project`? Less top-level commands like that. And therefore I'd keep the current state until we add a new one.

Reply:
`clean all` is good, yeah.

Reply:
So I add `all`?

Reply:
Yes.